### ISSUE TITLE
[a11y] Cannot view Locality record

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1420]
https://a11y-2-2-2-2.test.specifysystems.org/specify/view/locality/10695/

username: collection
db: saiab

![image](https://user-images.githubusercontent.com/37256050/167203978-43a1c51b-8481-4ec5-b666-b9f8e5f0548c.png)

**Comment by grantfitzsimmons** [comment]
![image](https://user-images.githubusercontent.com/37256050/167204251-34478609-2e5b-400a-8ddf-9f0615d48ad7.png)

(not the same locality) This is how their forms should appear
